libpdb/client_config.go

```diff
@@ -0,0 +1,36 @@
+package libpdb
+
+import (
+	"encoding/json"
+	"github.com/ryscheng/pdb/common"
+	"io/ioutil"
+  "time"
+)
+
+type ClientConfig struct {
+	*common.CommonConfig `json:"-"`
+
+  // How often should Writes be made to the server
+	WriteInterval  time.Duration
```

Comment:
I guess since you're using anonymous fields, this is something that can be easily moved around, so nbd